tests/unit/utils.unitTest.ts

```diff
@@ -15,15 +15,15 @@ describe("utils", () => {
   });
 
   describe("when filtering dynamoDB streams events", () => {
-    it("should filter correctly cancelled tests", async () => {
+    it("should filter correctly cancelled tests", () => {
```

Comment:
What's the reason these are no longer async?

Reply:
This was answered on the original ticket that went into CB2-303030.

You put the following on comment on there https://github.com/dvsa/cvs-tsk-cert-gen-init/pull/62#discussion_r1466205815. 

Which is why it was removed.